src/pages/groupComparison/ClinicalData.tsx

```diff
@@ -191,7 +192,14 @@ export default class ClinicalData extends React.Component<
 
     private tableDataStore = new ClinicalDataEnrichmentStore(
         () => {
-            return this.props.store.clinicalDataEnrichmentsWithQValues.result;
+            // exclude survival months and status from the data store
```

Comment:
are we sure we don't want to filter these out higher up in our application?  are they useable elsewhere, just not in this table?   we should include in the comment WHY we are excluding them.

Reply:
@alisman good point - the tests in the clinical tab are not appropriate for survival data according to the biostats team

I think for now we only want to exclude it from the comparison tab but we might want to exclude it from the plots tab as well in the future

Reply:
@alisman Added more comments to explain why: 
```
// ClinicalData is only used in Group Comparison Page and Group Comparison Tab
// And survival related attributes are used to create KM plots in survival tab
// We are doing the proper statistical tests in the survival tab and the used tests for the clinical tab are not appropriate for that type of data
// There is no need to still keep them in clinical tab
// So, exclude them from the data store
```